purchase_open_qty_stock_uom/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Purchase Open Qty Stock Uom",
+    "summary": "Add Qty to Receive (Stock UoM) field.",
+    "version": "16.0.1.0.0",
+    "author": "Toshikimi Shigenobu",
+    "website": "https://www.quartile.co",
+    "category": "Purchases",
+    "depends": [
+        "purchase_open_qty",
+    ],
+    "data": ["views/purchase_view.xml"],
```

Comment:
```suggestion
    "data": ["views/purchase_order_views.xml"],
```
Maybe this file will be changed, it should follow OCA's basic rules.